packages/vsce/src/utils/commandUtils.ts

```diff
@@ -13,6 +13,148 @@ import { TreeView } from "vscode";
 import { IResourceMeta } from "../doc";
 import { CICSRegionsContainer, CICSResourceContainerNode } from "../trees";
 import { IResource } from "@zowe/cics-for-zowe-explorer-api";
+import { IProfileLoaded } from "@zowe/imperative";
+import { Gui, ZoweExplorerApiType, ZoweVsCodeExtension } from "@zowe/zowe-explorer-api";
+import * as vscode from "vscode";
+import { CICSLogger } from "./CICSLogger";
+import { ProfileManagement } from "./profileManagement";
+
+/**
+ * Checks if a profile supports a specific type of connection
+ *
+ * @param profile - The profile to check
+ * @param connectionType - The type of connection to check for (ZoweExplorerApiType.Uss or ZoweExplorerApiType.Jes)
+ * @returns True if the profile supports the specified connection type, false otherwise
+ */
+export function doesProfileSupportConnectionType(profile: IProfileLoaded, connectionType: ZoweExplorerApiType): boolean {
+  const explorerApi = ZoweVsCodeExtension.getZoweExplorerApi();
+
+  try {
+    switch (connectionType) {
+      case ZoweExplorerApiType.Uss:
+        explorerApi.getUssApi(profile);
+        break;
+      case ZoweExplorerApiType.Jes:
+        explorerApi.getJesApi(profile);
+        break;
+      default:
+        return false;
+    }
+    return true;
+  } catch {
+    CICSLogger.error(`Profile ${profile.name} does not support connection type ${connectionType}`);
```

Comment:
I don't think this needs to be an error debug, because every time I run the command to show bundle directory, this runs through every available profile to check whether they support USS.

I think this should be debug - disabled by default. Otherwise the log will be overrun with "business as usual - this CICS connection isn't a USS connection"
```suggestion
    CICSLogger.debug(`Profile ${profile.name} does not support connection type ${connectionType}`);
```